packages/mgt-components/src/components/mgt-login/mgt-login.ts

```diff
@@ -79,6 +82,28 @@ export class MgtLogin extends MgtTemplatedComponent {
   })
   public userDetails: IDynamicPerson;
 
+  /**
+   * determines if presence is shown for logged in user
+   * defaults to false
+   * @type {boolean}
+   */
+  @property({
+    attribute: 'show-presence',
+    type: Boolean
+  })
+  public showPresence = false;
```

Comment:
Docs PR for these new properties exist?

Reply:
This has been added and is linked from this PR